keycert/base/src/main/java/se/swedenconnect/signservice/certificate/base/AbstractCaEngineKeyAndCertificateHandler.java

```diff
@@ -106,6 +111,20 @@ protected List<X509Certificate> issueSigningCertificateChain(
       final AbstractCertificateModelBuilder<? extends AbstractCertificateModelBuilder<?>> certificateModelBuilder =
           this.createCertificateModelBuilder(signingKeyPair.getPublicKey(), certificateNameModel);
 
+      // Add QC declarations
+      SigningCertificateRequirements certificateRequirements = signRequest.getSigningCertificateRequirements();
+      checkCertificateType(certificateRequirements);
```

Comment:
This should already have been done. Remove.

Reply:
We agreed to keep the check in to ensure that this function can't be called in violation with configuration constraints.